tests/src/system/basic/WskBasicTests.scala

```diff
@@ -166,6 +168,86 @@ class WskBasicTests
             stdout should not include regex(""""key": "xxx"""")
     }
 
+    it should "create, and get a package to verify annotation parsing" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "packageAnnotations"
+
+            assetHelper.withCleaner(wsk.pkg, name) {
+                (pkg, _) =>
+                    pkg.create(name, annotations = getValidJSONTestArgInput)
+            }
+
+            val stdout = wsk.pkg.get(name).stdout
+            assert(stdout.startsWith(s"ok: got package $name\n"))
+
+            val resJSON = stdout.substring(stdout.indexOf("\n") + 1).parseJson.asJsObject
+            assert(resJSON.fields("annotations") == getValidJSONTestArgOutput)
```

Comment:
I suggest you do not use asserts and instead use scalatest matchers (which we use throughout this suite and others for examples, or http://www.scalatest.org/user_guide/using_matchers). They generate better error messages in general and read better.

e.g. `resJson.fields("annotations") shouldBe(getValidJSONTestArgOutput)` and if the two end up not matching you'll get a nice error messages showing the two sides of the comparison. 

Also since you use this more than once, suggest we make it a utility in `Wsk`:
` +            val resJSON = stdout.substring(stdout.indexOf("\n") + 1).parseJson.asJsObject `
In fact there are at least two cases of this in `Wsk.scala` already to consolidate then.